In [ ]:
new=4.80
old= 6.83
percentage=(new-old)/old*100
percentage

-13.26530612244898

**MCMC TRAINED AND TESTED ON MCMC DATA**

*MCMC_MCMC-(No Augmentation)*\
$\mathrm{Rel-MLE}_{I}=5.79\%$ \ 
Relative Error near threshold:$0$--$10$ \

*MCMC_MCMC-(Augmentation)*\
$\mathrm{Rel-MLE}_{I}=4.56\%$\
Relative Error near threshold:$0-10\%$ 
Percentage change: $-21.24\% $

**LHS TRAINED AND TESTED ON LHS DATA**

*LHS_LHS-(No Augmentation)*
$\mathrm{Rel-MLE}_{I}=7.84\%$ 
Relative Error near threshold:$0-50\%$

*LHS_LHS-(Augmentation)*    
$\mathrm{Rel-MLE}_{I}=6.80\%$\
Relative Error near threshold:$2-20\%$ 
Percentage change: $-13.26\% $

**UNIFORM RANDOM TRAINED AND TESTED ON UNIFORM RANDOM DATA**

*UNIFORM-UNIFORM (No Augmentation)*\
$\mathrm{Rel-MLE}_{I}=6.83\%$ 
Relative Error near threshold:$0-50\%$

*UNIFORM-UNIFORM-(Augmentation)*\     
$\mathrm{Rel-MLE}_{I}=4.80\%$\
Relative Error near threshold:$0-50\%$ 
Percentage change: $-13.26\% $\


**MCMC TRAINED MODEL ON OUT OF SAMPLE DATA**

*MCMC_LHS (No Augmentation)*\
$\mathrm{Rel-MLE}_{I}=6.72\%$      
Relative Error near threshold:$0-20$

*MCMC_LHS (Augmentation)*\
$\mathrm{Rel-MLE}_{I}=5.62\%$\
Relative Error near threshold:$0-10$
Percentage change: $-16\% $\

*MCMC_Random (No Augmentation )*    
No Augmentation  $\mathrm{Rel-MLE}_{I}=6.81\%$\
Relative Error near threshold:$2-20%$

*MCMC_Random (Augmentation )*   
$\mathrm{Rel-MLE}_{I}=5.79\%$\ 
Relative Error near threshold:$0-10%$
Percentage change= $-13.26%$\

**LHS TRAINED MODEL ON MCMC TEST DATA**

*LHS_MCMC (No Augmentation )*\
$\mathrm{Rel-MLE}_{I}=9.04\%$\     
Relative Error near threshold:$0-50\%$

*LHS_MCMC (Augmentation )*\
$\mathrm{Rel-MLE}_{I}=7.84\%$\
Relative Error near threshold:$0-25\%$
Percentage change: $-13.27\% $\

**UNIFORM RANDOM TRAINED MODEL ON MCMC TEST DATA**

*UNIFORM_MCMC (No Augmentation)*\
$\mathrm{Rel-MLE}_{I}=9.57\%$ \ 
Relative Error near threshold:$0-50\%$    

*UNIFORM_MCMC ( Augmentation)*\
$\mathrm{Rel-MLE}_{I}=6.31\%$\
Relative Error near threshold:$0-25$
Percentage change: $-34.06\% $\

